src/js/modules/add_photo.js

```diff
@@ -12,6 +12,17 @@ let formState = {
   photo: null,
 };
 
+function attachImgs() {
+  const addBtn = document.querySelector(".btn__add");
+  const iconSpan = document.createElement("img");
+  addBtn.appendChild(iconSpan);
+  iconSpan.src = require("../../icons/png/016-add.png");
```

Comment:
You could also import the image at the top, you wanted to. I use inline require just as a personal preference.